<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <test thread-count="5" parallel="tests" name="Test(failed)" verbose="0">
    <classes>
      <class name="com.atmecs.api_authentication.test.DeleteRecord">
        <methods>
          <include name="deleteRecord"/>
        </methods>
      </class> <!-- com.atmecs.api_authentication.test.DeleteRecord -->
      <class name="com.atmecs.api_authentication.test.UpdateRecord">
        <methods>
          <include name="updateUser" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.atmecs.api_authentication.test.UpdateRecord -->
      <class name="com.atmecs.api_authentication.test.PatchRecord">
        <methods>
          <include name="updateUser" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.atmecs.api_authentication.test.PatchRecord -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
